React/Views/RCTShadowView.m

```diff
@@ -281,10 +281,6 @@ - (void)layoutWithMinimumSize:(CGSize)minimumSize
 
   RCTAssert(!YGNodeIsDirty(yogaNode), @"Attempt to get layout metrics from dirtied Yoga node.");
 
-  if (!YGNodeGetHasNewLayout(yogaNode)) {
```

Comment:
I am a bit concerned about this change because it relies on current implementation details of Yoga. Yoga is an independent library and its implementation can change any time. And, if we remove change, why should not we remove the next line that resets the flag?

Reply:
@shergin Thanks for review. It's true that we should not rely details on third library. The reason I don't remove the line of resets the flag is maybe other code may query `YGNodeGetHasNewLayout` of this `node`, but actually, we don't have this check currently, IMO, maybe keep this reset is ok, we can keep the same way, when we reset it, it means we already layout the views.
So what's your opinion? 🤔 We can remove `hasNewLayout` check? Or just to abandon this `PR`?